lib/fastlane/plugin/ddg_apple_automation/helper/git_helper.rb

```diff
@@ -77,6 +77,31 @@ def self.assert_branch_has_changes(release_branch)
 
         changed_files.any?
       end
+
+      def self.latest_release(repo_name, prerelease, platform = nil, github_token)
+        client = Octokit::Client.new(access_token: github_token)
+
+        current_page = 1
+        page_size = 25
```

Comment:
Happy to change this to something else, but I feel that 25 should be enough to always find the latest public release for a given platform.